lib/auth/person.ex

```diff
@@ -227,12 +245,9 @@ defmodule Auth.Person do
       # existing person
       ep ->
         merged = Map.merge(AuthPlug.Helpers.strip_struct_metadata(ep), person)
-
-        {:ok, person} =
-          changeset(%Person{id: ep.id}, merged)
-          |> Repo.update()
-
-        person
+        {:ok, person} = Repo.update(changeset(%Person{id: ep.id}, merged))
```

Comment:
IMO, using pipe here as previous would make it clearer due to the amount of language clutter on one line

Reply:
Yeah, I completely agree. I only changed it because Credo/SourceLevel was complaining. 🙄